subxt/src/config/mod.rs

```diff
@@ -34,7 +35,6 @@ pub trait Config: 'static {
         + Member
         + serde::de::DeserializeOwned
         + Default
-        + AtLeast32Bit
```

Comment:
why removed? I guess that trait comes from substrate

Reply:
Yeah; I basically removed various bits that we didn't actually make use of or care about on the Subxt side. Like, if somebody configures the type wrong (whether it's because it's not 32bit or simply just not what the chain is configured with) it won't work either way, so I didn't see the point of the extra check anyway